src/lib/paths.js

```diff
@@ -54,7 +54,10 @@ const pathUtils = {
               packageJsonPath
             );
 
-            if (dependencies['flow-bin'] || devDependencies['flow-bin']) {
+            if (
+              (dependencies && dependencies['flow-bin']) ||
```

Comment:
Hey @stevehollaar - Awesome fix, thank you so much! Just one little note, we could simplify this change by adding default values on [L53](https://github.com/ImmoweltGroup/flow-mono-cli/blob/master/src/lib/paths.js#L53). For example instead of the current implementation

```js
const {dependencies, devDependencies} = await file.readJson(
  packageJsonPath
);
```

it could look like

```js
const {dependencies = {}, devDependencies = {}} = await file.readJson(
  packageJsonPath
);
```

This way we would fix this common error for future cases where we need to access dependencies or the like. If you would change this little nitpick I would be super happy to merge this PR, again, thanks a lot! ❤️

Reply:
Hey @Inkdpixels. Thanks for the library! Pushed the suggested changes (agree that's cleaner). Let me know if you need anything else changed 😄